Create CODE_OF_CONDUCT.md

tharunc opened the pull request:
CODE_OF_CONDUCT.md has been created.

tharunc committed:
Create CODE_OF_CONDUCT.md

CODE_OF_CONDUCT.md has been created.

arpansac commented:
Hey I went through it, just 2 things, it
- It needs to mention that this code of conduct is for the developers contributing to Commudle
- It should mention the links to the privacy policy and T&C which are present on the official website
- It's link should be mentioned in the readme

arpansac reviewed (changes requested):
Hey I went through it, just these things, it

It needs to mention that this code of conduct is for the developers contributing to Commudle
It should mention the links to the privacy policy and T&C which are present on the official website
It's link should be mentioned in the readme

tharunc commented:
Sure, but can you please mention the Line for the formatting to be done.

arpansac commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
```
For Commudle

arpansac commented on CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,75 @@
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
```
Add a section for T&C, Privacy Policy

arpansac reviewed (commented):
> Sure, but can you please mention the Line for the formatting to be done.

Sure, added 2 comments, for the 3rd point please put it just below the contribution guidelines link in the Readme

tharunc committed:
Updated CODE_OF_CONDUCT.md as a part of DWOC

I have included developers contribution for Commudle.
I have included sections for T&C and Privacy Policy from the official Commudle website.

tharunc committed:
Update CODE_OF_CONDUCT.md

tharunc committed:
Update README.md

tharunc committed:
Update README.md

arpansac commented:
Cool, will review and merge by tomorrow

tharunc commented:
Kindly add LEVEL tag for this merge.
